### ISSUE TITLE
How do you launch correctly the server?

### ISSUE THREAD

**Issue by valepu** [shibbir/generator-angular2-typescript#4]
Hello,
I have installed the generator with Foundation and SystemJS, but when i launch "npm start" then a browser page opens i only see "Loading..." on the screen.
If i open the console i see this error

```
Error: patchProperty/desc.set/wrapFn@http://localhost:3000/node_modules/zone.js/dist/zone.js:769:27
    Zone</ZoneDelegate</ZoneDelegate.prototype.invokeTask@http://localhost:3000/node_modules/zone.js/dist/zone.js:356:24
    Zone</Zone</Zone.prototype.runTask@http://localhost:3000/node_modules/zone.js/dist/zone.js:256:29
    ZoneTask/this.invoke@http://localhost:3000/node_modules/zone.js/dist/zone.js:423:29

    Error loading http://localhost:3000/foundation-sites/dist/foundation.css as "foundation-sites/dist/foundation.css" from http://localhost:3000/app/app.component.js
```

So i am wondering if i'm missing something to run the application

I am using Windows 10, if it changes something

**Comment by shibbir** [comment]
Fixed with [#e83e900](https://github.com/shibbir/generator-angular2-typescript/commit/e83e9009534c23b786ee2de9b216c87cea83f413)

New package with this fix is published in [NPM](https://www.npmjs.com/package/generator-angular2-typescript). So please install the package again. That should fix your problem.

**Comment by valepu** [comment]
Hi,
I can confirm it works now, thanks
